DEVELOPMENT.md

```diff
@@ -0,0 +1,19 @@
+# ImageDePHI
+
+This project has been funded in whole or in part with Federal funds from the National Cancer Institute, National Institutes of Health, Department of Health and Human Services, under Contract No. 75N91022C00033
+
+## To Run as a CLI
+
+`pip install -r requirements.txt`
+
+`python imagedephi.py <directory of input images> <directory to store redacted images>`
```

Comment:
I don't think we want developers doing any of this now.

For local development, we probably want them to:
* create a virtualenv (we should assume they know how to dot his already, but we could maybe link to docs)
* run `pip install -e .`
* execute with `imagedephi`

For running tests, I think we can just tell people:
* `pip install tox`
* `tox`

If we explicitly document how to run tests, I think we should actually remove the `requirements.txt` file, since it only contains `tox`.

Reply:
updated in a89e9c9d71a8b2ec6a0ddaa81f819ffd05d1f6e7